app/src/rgb_underglow.c

```diff
@@ -58,7 +57,7 @@ static struct led_rgb hsb_to_rgb(struct zmk_led_hsb hsb) {
     double r, g, b;
 
     uint8_t i = hsb.h / 60;
-    double v = hsb.b / ((float)BRT_MAX);
+    double v = hsb.b / ((float)CONFIG_ZMK_RGB_UNDERGLOW_BRT_MAX);
```

Comment:
I actually think this change is wrong, as it would turn up the brightness indirectly by converting it back to RGB, so I'm changing it back to `BRT_MAX` (100).